release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4040,4 +4040,23 @@ $ oc adm release info 4.12.32 --pullspecs
 [id="ocp-4-12-32-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
```

Comment:
```suggestion
To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
Consider omitting `for instructions`. It's already clear from the context and not needed, imo.